src/utils/modules.js

```diff
@@ -27,6 +27,47 @@ const validCommandsString = validCommands.map(command => `${command} ${commandTr
 
 const isValidCommand = command => validCommands.includes(command);
 
-const getModule = moduleShortcut => availableModules[moduleShortcut];
+const escapeRegExp = str => str.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
 
-export { getModule, isValidCommand, validCommandsString };
+const getModuleShortcutMatch = moduleShortcut => {
+  for(const key of Object.keys(availableModules)) {
+    const patternElements = key.split(/{[a-zA-Z_\[\]\-]+}/);
```

Comment:
Liten sanity check, siden jeg ikke har helt oversikt over formatet på input:

Separatoren her er en streng, mellom bokstavelig matchede krøllparenteser, av vilkårlig lengde, som inneholder bokstaver, _, [, ], og -. Strengene som matches blir stedene hvor det klippes i input-strengen for å generere output-arrayet, og blir ikke med i output. Er dette meningen?

Trivielt eksempel, for kontekst:
```javascript
var str = "How are you doing today?";
var res = str.split(" ");
# output: ["How", "are", "you", "doing", "today?"]
```

Eksempel med denne regexen:
```javascript
var str = "asd {asdaA[F[FGs---_sdf]sdgf}okokok";
var res = str.split(/{[a-zA-Z_\[\]\-]+}/);
# output: ["asd ", "okokok"]
```

Reply:
Dette kan f.eks. testes i praksis på [Regex101](https://regex101.com/).